gatsby/content/blog/2022/01/2022-01-18-synapse-1.50-released.mdx

```diff
@@ -0,0 +1,45 @@
+
+We've recently started work to tackle this issue in the form of [MSC3202](https://github.com/matrix-org/matrix-doc/pull/3202). A first part of implementing this MSC (allowing application services to masquerade as specific devices) has landed in this release of Synapse; work is still ongoing towards a full implementation, so watch this space!
+
+
+## Improved reliability on device list updates
+
+While working on this release, we identified a long-standing bug that could prevent Synapse from sending device lists update over federation if the server had a high number of active users and/or users with a lot of devices connected to their account.
+
+This bug was introduced back in [Synapse 1.0.0](https://github.com/matrix-org/synapse/releases/tag/v1.0.0), and meant that the homeserver would miss some device list updates when communicating with other homeservers if the amount of updates to send was too high. In practice, this means users on remote homeservers could see outdated device information for other users (including outdated device verification statuses).
+
+Synapse 1.50 includes a fix to this bug. This should contribute towards making the propagation of device list updates more reliable.
+
+
+## Everything else
+
+This release introduces support for [MSC3419](https://github.com/matrix-org/matrix-doc/pull/3419), which allows guest users to send arbitrary state events into a room. This will be especially useful to the ongoing work on [group VoIP calls](https://github.com/matrix-org/matrix-doc/pull/3401), which involves having users send new state events into the room to signal their participation in a call.
+
+We've also stabilised identifiers for cross-signing and fallback keys now that [MSC1756](https://github.com/matrix-org/matrix-doc/pull/1756) and [MSC2732](https://github.com/matrix-org/matrix-doc/pull/2732) have been merged into the Matrix spec.
+
+On the documentation side of things, the [page](https://matrix-org.github.io/synapse/v1.50/turn-howto.html) on setting up and configuring a TURN server has been updated to feature instructions on how to deal with NATs. This is a much welcome addition as configuring TURN is something a lot of Synapse admins struggle with!
+
+Please see the Synapse [release notes](https://github.com/matrix-org/synapse/blob/v1.50.0/CHANGES.md) for a complete list of changes in this release.
+
+Synapse is a Free and Open Source Software project, and we'd like to extend our thanks to everyone who contributed to this release, including [Dirk Klimpel](https://github.com/dklimpel), [Donny Johnson](https://github.com/itchychips) and [AndrewFerr](https://github.com/AndrewFerr).
+
+> Note: An issue preventing client logins ([#11763](https://github.com/matrix-org/synapse/issues/11763)) was identified immediately following the release of Synapse 1.50.0. We released [Synapse 1.50.1]((https://github.com/matrix-org/synapse/releases/tag/v1.50.0)) the same day with a fix for this issue.
```

Comment:
The post is about the 1.50 release as a whole (vs 1.50.0), which includes 1.50.1.

More generally, I think we're going to move towards blogging more about minor versions than patch versions, since we don't usually blog for patch version bumps (and the first one is the one with most user-facing changes).

Plus ime it's easy enough for ops teams to figure out from changelog and release tags that there's a bugfix version for this release.